docs/self-managed/setup/guides/identity-oracledb.md

```diff
@@ -0,0 +1,141 @@
+---
+id: identity-oracledb
+title: "Using OracleDB with Identity for OIDC"
```

Comment:
:wrench: I guess we should change the title to match the more generic content, i.e. not all Oracle specific?

Reply:
I made the title more generic